src/models/icons/iconPack.ts

```diff
@@ -9,4 +9,5 @@ export enum IconPack {
   Redux = 'react_redux',
   Vue = 'vue',
   Vuex = 'vue_vuex',
+	VuePinia = 'vue_pinia',
```

Comment:
```suggestion
	Pinia = 'pinia',
```

Reply:
Is it better to name it just "pinia" or rather "vue_pinia"? I think both are fine.